packages/contracts/package.json

```diff
@@ -47,9 +47,16 @@
   },
   "dependencies": {
```

Comment:
Please add to `devDependencies` instead of adding to `dependencies` (`yarn add <package name> --dev`).

For this package, you will need to install the following dev dependencies:
- `@ethersproject/abi`
- `@nomiclabs/ethereumjs-vm`
- `@ethersproject/bytes`
- `@ethersproject/providers`
- `@ethersproject/abi`
- `@types/node`

You shouldn't have to install any dependencies other than these.